docs/Developers/Newcomers_Guide.rst

```diff
@@ -298,7 +298,63 @@ Ask a developer to check if it should be reclassified as `low` difficulty.
       please check the following link:
       `PEP8 Style Guide for Python code <https://www.python.org/dev/peps/pep-0008/>`_
 
-Step 3. Creating a Fork and Testing Your Changes
+Step 3. Verifying the Issue Exists
+----------------------------------
+
+You must ensure that the issue you would like to work on exists. You should
```

Comment:
add something like `You must ... because it is possible that it has already been fixed.`

Reply:
Reason for this is that sometimes it happens that some issue gets resolved due to a PR for a different issue.